x/incentives/keeper/distribute_test.go

```diff
@@ -92,6 +93,97 @@ func (suite *KeeperTestSuite) TestDistribute() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestDistributeToCLPools() {
+	fiveKRewardCoins := sdk.NewInt64Coin(defaultRewardDenom, 5000)
+	fifteenKRewardCoins := sdk.NewInt64Coin(defaultRewardDenom, 15000)
+
+	coinsToMint := sdk.Coins{sdk.NewInt64Coin(defaultRewardDenom, 1000000)}
+
+	tests := map[string]struct {
+		// setup
+		numPools int
+
+		// expected
+		expectErr             bool
+		expectedDistributions sdk.Coins
+	}{
+		"valid case: one poolId and gaugeId": {
+			numPools:              1,
+			expectErr:             false,
+			expectedDistributions: sdk.Coins{fiveKRewardCoins},
+		},
+
+		"valid case: multiple gaugeId and poolId": {
+			numPools:              3,
+			expectErr:             false,
+			expectedDistributions: sdk.Coins{fifteenKRewardCoins},
+		},
+	}
+
```

Comment:
we also need test cases that tests mixture of providing test variables of non cl gauge + cl gague

Reply:
hmm, i tried testing this but even with non-cl gauge the distribute functions doesnot error and works as expected. This was to make things backwards compatible

Reply:
Right, but shouldn't that be a test case still? Showing that a test case that mixes cl and non cl gauge does not error and works as expected?

Reply:
yup i ended up adding this test case here https://github.com/osmosis-labs/osmosis/pull/4526/files#diff-d5967ddef40ae47ef4140b9c47026cac85b85d9d04173b271c6c23dd182abdbcR153